### ISSUE TITLE
Add optional VNC server installation

### ISSUE THREAD

**Issue by tracer0tong** [tracer0tong/android-emulator#8]
Optionally entrypoint could install http://www.onaips.com/wordpress/?page_id=60 (https://github.com/oNaiPs/droidVncServer) to emulator.

**Comment by taesiri** [comment]
Having a VNC server on the emulator is good, but there is a good project called AndroidViewClient. It has a tool called [Culebra GUI](https://github.com/dtmilano/AndroidViewClient/wiki/Culebra-GUI) for interacting with emulator over adb.

Another option is sharing X11 socket with the Host to render emulator window in the host.

**Comment by tracer0tong** [comment]
@taesiri I've seen this project. As I understand it based on standard Monkey Runner. It means that developers don't need any additional components in container. Initially I've made this container for server emulation of multiple Android devices and don't think about VNC. Also I've used Monkey Runner for tests, that's why I've postponed VNC for long time. Anyways if you have any suggestions, I'm always happy to see pool requests or request for changes.
